app/controllers/users_controller.rb

```diff
@@ -51,7 +51,7 @@ def reorder_call_list
   private
 
   def user_params
-    params.require(:user).permit(:name, :email, :_role)
+    params.require(:user).permit(:name, :email, :role)
```

Comment:
If this switches enums from ints to strings, I'd very much appreciate commands to run in the rails console that will let us cleanly migrate things over to the new datatypes.

e.g. 
```
User.all.each do |user|
  if user[:_role] == 'admin'
    user.role = 'admin'
  elsif ...
end
```

That said having enums as strings is sort of annoying! Breaks the pattern from standard activerecord enums. Kinda sucks that every implementation of this is bad. I'm good with this if it means we get away from mongoid enum though.